### DISCUSSION TITLE
Installation storage requirements

### DISCUSSION THREAD

**Discussion by justinfreitag** [nzymedefense/nzyme#487] [Questions] [answered]
Hi all, I'm setting up nzyme of a Pi 4 4GB. Can anyone provide some advice on the minimum and/or typical storage requirements of the the Postgres DB as well as Graylog. For Graylog I'd be looking at storing between 2-4 weeks of data. I've got 3 radios.

**Comment by lennartkoopmann** [comment, marked as answer]
Not sure on the Graylog requirements because it will depend at lot on your WiFi environment. I suggest just sending a few hours of data and use that to calculate your total requirement?

For nzyme, the database will not need much space and it will also not growing over time because it automatically retention cleans all data. A few hundred MB if you want to be on the absolute safe side.
